DTML.EduBot/Dialogs/GreetingDialog.cs

```diff
@@ -14,14 +14,13 @@ public partial class RootDialog : QnaLuisDialog<object>
         [LuisIntent("Greeting")]
         public async Task Greeting(IDialogContext context, LuisResult result)
         {
-            string botresponse = BotPersonality.GetRandomGreeting(); 
-            await context.PostAsync(botresponse);
+            await ConvertAndPostResponse(context, BotPersonality.GetRandomGreeting());
```

Comment:
I was thinking of translating every time except when the lessons are going on. That way a newbie who is just learning English could come up and use the bot to start from scratch.